src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/system_information.html.twig

```diff
@@ -27,261 +27,275 @@
-                                {% endif %}
-                            {% endfor %}
-                        </ul>
-                    {% endif %}
-                    {% if requirements.failOptional is defined %}
-                        <p>
-                        <strong>{{ 'Optional parameters:'|trans }}</strong>
-                        {% if requirements.failOptional == false %}
-                            <span class="text-success">{{ 'OK'|trans({}, 'Admin.Advparameters.Notification') }}</span>
-                            </p>
-                        {% else %}
-                            <span class="text-danger">{{ 'Please fix the following error(s)'|trans({}, 'Admin.Advparameters.Notification') }}</span>
-                            </p>
-                            <ul>
-                                {% for key, value in requirements.testsOptional %}
-                                    {% if 'fail' == value %}
-                                        <li>{{ requirements.testsErrors[key] }}</li>
-                                    {% endif %}
-                                {% endfor %}
-                            </ul>
-                        {% endif %}
-                    {% endif %}
-                </div>
-            </div>
+    <div class="card" id="checkConfiguration">
+      <h3 class="card-header">
+        <i class="material-icons">info_outline</i> {{ 'Check your configuration'|trans }}
+      </h3>
+      <div class="card-block">
+        <div class="card-text">
+          <p>
+            <strong>{{ 'Required parameters:'|trans }}</strong>
+            {% if requirements.failRequired == false %}
+            <span class="text-success">{{ 'OK'|trans({}, 'Admin.Advparameters.Notification') }}</span>
+          </p>
+          {% else %}
+          <span class="text-danger">{{ 'Please fix the following error(s)'|trans({}, 'Admin.Advparameters.Notification') }}</span>
+          </p>
+          <ul>
+            {% for key, value in requirements.testsRequired %}
+            {% if 'fail' == value %}
+            <li>{{ requirements.testsErrors[key] }}</li>
+            {% endif %}
+            {% endfor %}
+          </ul>
+          {% endif %}
+          {% if requirements.failOptional is defined %}
+          <p>
+            <strong>{{ 'Optional parameters:'|trans }}</strong>
+            {% if requirements.failOptional == false %}
+            <span class="text-success">{{ 'OK'|trans({}, 'Admin.Advparameters.Notification') }}</span>
+          </p>
+          {% else %}
+          <span class="text-danger">{{ 'Please fix the following error(s)'|trans({}, 'Admin.Advparameters.Notification') }}</span>
+          </p>
+          <ul>
+            {% for key, value in requirements.testsOptional %}
+            {% if 'fail' == value %}
+            <li>{{ requirements.testsErrors[key] }}</li>
+            {% endif %}
+            {% endfor %}
+          </ul>
+          {% endif %}
+          {% endif %}
```

Comment:
This is so unreadable :(

Reply:
The IDE looks to be having a hard formatting the file, I will try to do it manually

Reply:
I found an unclosed p tag in the mail info, I re-wrote it and simplified it a bit

Reply:
Already way better :)